src/main/kotlin/com/blincke/commune_api/controllers/FriendsController.kt

```diff
@@ -83,7 +83,7 @@ class FriendsController(
         )
         userService.runAuthorized(userId, principal) {
             AppLoggerFactory.getLogger(this).debug("From user with id ${it.firebaseId}")
-            ResponseEntity.ok(friendService.getSuggestedFriendsForUser(it, page, pageSize)
+            ResponseEntity.ok(friendService.getSuggestedFriendsForUser(it, queryStr, page, pageSize)
```

Comment:
Oh yeah, I just added a quick query string to get suggested friends. Sorry that's a bit out of scope.

Reply:
I'd like to discuss the nature of suggested friends. People certainly need a way to see and add potential friends, but this seems a bit different from FB's "suggested friends" feature. I'd like to drill down on what is entailed in our implementation and what we're aiming for before launch